src/playback/summary.jsx

```diff
@@ -153,6 +155,30 @@ class PlaybackSummary extends React.Component {
     await this.update();
   }
 
+  tooltipContent(score, encoding, platform, size) {
+    if (score === lookupType['0'])
```

Comment:
instead of 

    if (score === lookupType['0'])

use

    if (score === 'pass')

You may want to declare constants,

```
const PASS = 'pass';
const BAD = 'bad';
const FAIL = 'fail';
const lookupType = {
  '0': PASS,
  '1': BAD,
  '2': FAIL,
};
```

and then 

    if (score === PASS)

but that is up to you:  The string constants are limited to this file, so do not require boilerplate yet.